### DISCUSSION TITLE
Parameter to turn off logging errors

### DISCUSSION THREAD

**Discussion by smsajjadzaidi** [kevinzg/facebook-scraper#789] [Ideas]
Hi, is there any optional parameter that I can use to turn off facebook scraper from logging errors? It is cluttering our sentry. If there is not any such parameter yet, then maybe you guys could add it?

**Comment by neon-ninja** [comment]
No errors are logged by default, try comment out your `enable_logging` call?

**Reply by smsajjadzaidi** [reply]
is this parameter supposed to be passed in `get_profile()` call?

**Reply by neon-ninja** [reply]
No. Maybe it would help if you posted your code

**Reply by smsajjadzaidi** [reply]
I am just calling `FacebookScraper().get_profile(account='some-username')` and I get errors like `data-ft attribute not found` `Friend_count extraction failed` etc.

**Reply by neon-ninja** [reply]
Try:

```python
import logging
logging.getLogger("facebook_scraper").propagate = False
```

**Reply by smsajjadzaidi** [reply]
thanks
